docs/en/operations/system-tables.md

```diff
@@ -1335,4 +1335,95 @@ Columns:
 
 If the storage policy contains more then one volume, then information for each volume is stored in the individual row of the table.
 
+## system.quotas {#system_tables-quotas}
+Contains information about [quotas](quotas.md).
+
+Columns:
+-  `name` ([String](../sql-reference/data-types/string.md)) — Quota name.
+-   `id` ([UUID](../sql-reference/data-types/uuid.md)) — Quota ID.
+-   `storage`([String](../sql-reference/data-types/string.md)) — Storage of quotas. Possible value: "users.xml" if a quota configured in the users.xml file, "disk" if a quota configured by an SQL-query.
+-   `key_type` ([Enum8](../sql-reference/data-types/enum.md)) — The type of key that the quota is tracked for. Values:
+    -   `'none' = 0` — All users share the same quota.
+    -   `'user name' = 1` — Connections with the same user name share the same quota.
+    -   `'ip address' = 2` — Connections from the same IP share the same quota.
+    -   `'client key' = 3` — Connections with the same key share the same quota. A key must be explicitly provided by a client. When using [clickhouse-client](../interfaces/cli.md), pass a key value in the `--quota-key` parameter, or use the `quota_key` parameter in the client configuration file. When using HTTP interface, use the `X-ClickHouse-Quota` header.
+    -   `'client key or user name' = 4` — Connections with the same `client key` share the same quota. If a key isn't provided by a client, the qouta is tracked for `user name`.
+    -   `'client key or ip address' = 5` — Connections with the same `client key` share the same quota. If a key isn't provided by a client, the qouta is tracked for `ip address`.
+-   `roles` ([Array](../sql-reference/data-types/array.md)([String](../sql-reference/data-types/string.md))) — Name of the [roles](../operations/access-rights.md#role-management) or users which share this quota.
```

Comment:
Don't use the word `share` in this text, because `sharing` is more likely about key, not about roles.
I've changed this field recently too.

Reply:
Yes, I change to "use".